src/api/app/models/review.rb

```diff
@@ -94,23 +97,31 @@ def check_initial
       errors.add(:unknown, 'no reviewer defined')
     end
 
-    if by_user && !User.find_by_login(by_user)
+    if by_user
```

Comment:
This surely can be written in various ways, eg. I would have preferred to have that specific validation in a separate validation method. Though I didn't find it important enough to require this for in PR.